scripts/travis-lava

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -x
+
+LAVA_CONTAINER_NAME=lava_server
+
+while getopts ":s" opt; do
+  case ${opt} in
+    s)
+      docker kill "${LAVA_CONTAINER_NAME}"
+      docker rm "${LAVA_CONTAINER_NAME}"
+      exit 0
+      ;;
+    * )
+      ;;
+  esac
+done
+
+
+docker run -p 5556:5556 -p 80:80 --name "${LAVA_CONTAINER_NAME}" lavasoftware/lava-server:latest &
+# wait for docker to start
+while [ ! $(docker inspect -f '{{.State.Running}}' "${LAVA_CONTAINER_NAME}") ]
+do
+    sleep 5
+    echo "Waiting for LAVA container to start"
+done
+docker_status=$(docker inspect -f '{{.State.Running}}' "${LAVA_CONTAINER_NAME}")
+while [ $docker_status = 'false' ]
```

Comment:
I think this second while loop is doing the exact same check as the first one above

Reply:
not really. When the container is absent (not even stopped) 'docker inspect' will throw exit code != 0. When the container is present but not running, exit code will be 0 but string reported by 'docker inspect' will tell the status. I wish it was more consistent but it isn't :(

Reply:
oh dang!